src/tokens/selectors.ts

```diff
@@ -419,7 +419,11 @@ export const cashOutTokensByNetworkIdSelector = createSelector(
       showZeroBalanceTokens,
   ],
   (tokens, showZeroBalanceTokens) =>
-    tokens.filter((tokenInfo) => showZeroBalanceTokens || tokenInfo.isCashOutEligible)
+    tokens.filter(
+      (tokenInfo) =>
+        (showZeroBalanceTokens || tokenInfo.balance.gt(TOKEN_MIN_AMOUNT)) &&
```

Comment:
could this be in sync with ms runtime? I am guessing no because cKES needs to be a zero balance token?